homeassistant/components/synology_dsm/sensor.py

```diff
@@ -79,7 +78,7 @@ def __init__(
         """Initialize the sensor."""
         self._api = api
         self.sensor_type = sensor_type
-        self._name = f"{BASE_NAME} {sensor_info[0]}"
+        self._name = f"{self._api.network.hostname} {sensor_info[0]}"
```

Comment:
Is `BASE_NAME` still used then ?

Reply:
No, the only thing BASE_NAME was used for in this module was to define all the names for the entities.

Reply:
wrt to using the network object, I thought it was better to get the object and be consistent with the rest of what the API provides.

Reply:
If not used anywhere else, it should be remove from const.